src/index/index.c

```diff
@@ -250,14 +281,25 @@ void Index_Construct
 	uint fields_count = array_len(idx->fields);
```

Comment:
line 265 above: `idx_options->lang = idx->language;`
how come we're accessing search "internals" unlike `RediSearch_IndexOptionsSetStopwords` were we have a function which sets stopwords, shouldn't we have a function to set the language ?